homeassistant/components/config/automation.py

```diff
@@ -16,15 +16,30 @@
 )
 from homeassistant.config import AUTOMATION_CONFIG_PATH
 from homeassistant.const import CONF_ID, SERVICE_RELOAD
+from homeassistant.core import callback
 from homeassistant.helpers import config_validation as cv, entity_registry
+from homeassistant.helpers.script import (
+    EVENT_SCRIPT_BREAKPOINT_HIT,
+    breakpoint_clear_all,
+    breakpoint_list,
+    breakpoint_set,
+    debug_continue,
+    debug_step,
+)
 
 from . import ACTION_DELETE, EditIdBasedConfigView
 
 
 async def async_setup(hass):
     """Set up the Automation config API."""
 
-    websocket_api.async_register_command(hass, websocket_automation_trace)
+    websocket_api.async_register_command(hass, websocket_automation_trace_get)
+    websocket_api.async_register_command(hass, websocket_automation_breakpoint_clear)
+    websocket_api.async_register_command(hass, websocket_automation_breakpoint_list)
+    websocket_api.async_register_command(hass, websocket_automation_breakpoint_set)
+    websocket_api.async_register_command(hass, websocket_automation_debug_continue)
+    websocket_api.async_register_command(hass, websocket_automation_debug_step)
+    websocket_api.async_register_command(hass, websocket_subscribe_breakpoint_events)
```

Comment:
I wonder if we should merge this with the breakpoint_set. That way the registered unsubscribe function can also clear the breakpoint.

Reply:
You might want to set / clear several breakpoints during a debug session though?

Reply:
Wouldn't it be easier to make each of those their own subscribe ? That way you are guaranteed they are cleaned up

Reply:
No, because we want to get updates also for breakpoints not set by the user, for example when single stepping.